src/main/java/io/ebean/bean/EntityBean.java

```diff
@@ -96,4 +96,9 @@
    */
   Object _ebean_getFieldIntercept(int fieldIndex);
 
+  /**
+   * Recalcs the bean. This is invoked by PersistRequestBean.
+   */
+  default void _ebean_recalc() {};
```

Comment:
This is almost '_ebean_prePersistDirty' ?

We ought to javadoc the difference with normal PrePersist ...

Reply:
How about `_ebean_onPersistTrigger()`